LEGO1/lego/legoomni/include/pizza.h

```diff
@@ -6,6 +6,8 @@
 #include "isleactor.h"
 #include "legostate.h"
 
+#include <climits>
```

Comment:
What is this header inclusion needed for? If we really need it we should make it limits.h instead of climate since that's what has been used so far.
(I'd certainly prefer the C++ versions in a new project but for consistency we shouldn't start using them)

Reply:
It was for a `INT_MIN` which is defined by windows.h, but not when targeting Linux.

Reply:
There are more similar occurrence that don't:t show up because of my incomplete windows.h wrapper.
Those will be addressed in time.